### PR TITLE
Create a basic page to display past scholarX.

### LINKED ISSUE #543: Create a basic page to display past scholarX.
**Is your feature request related to a problem? Please describe.**
- We had to create a page to display our past scholarx programs. 

**Describe the solution you'd like**
- This page should be a super simple one. Have a look at this: https://summerofcode.withgoogle.com/archive/
- This page has a hero section.
- Use hero section from the current scholarx page.
- after that there will be cards for each year sholarx program which looks like this.
Screenshot from 2020-02-27 19-39-08
- Link this card to the 2019 scholarx page.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #543 

## Goals

- This page should be a super simple one. https://summerofcode.withgoogle.com/archive/

## Approach

- Use hero section from the current scholarx page.
-  Create a card for 2019 scholarx.
- Link this card to the 2019 scholarx page.
  
### Preview Link

https://pr-549-sef-site.surge.sh/scholarx/archive/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/assets/css/argon.css [hunk 1]
```diff
@@ -7948,6 +7948,11 @@ button.bg-primary:focus
     background-color: #f4f5f7 !important;
 }
 
+.bg-secondary-light
+{
+    background-color: #f7f8fa !important;
+}
+
```
Comment (Gravewalker666) [inline src/main/webapp/assets/css/argon.css]: Don't change the argon.css file. If you need to add external CSS or Overwrite the argon styles use global-custom.css file.

File: src/main/webapp/scholarx/past-scholarx.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+            <div class="col-9 text-center">
+                <h1 class="mb-0 text-onelive">Take a look at our past</h1>
+                <p class="display-3 text-onelive font-weight-bolder">ScholarX</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-6">
+                <img class="card-img"
+                     src="images/card-image.jpg">
+                <div class="card" style="width: auto;">
```
Comment (Gravewalker666) [inline src/main/webapp/scholarx/past-scholarx.html]: Try to use bootstrap as much as you can and use CSS only by adding classes to the global-custom CSS file. Avoid using inline CSS styles.

**Review by Gravewalker666 — DISMISSED** [review]

File: src/main/webapp/scholarx/past-scholarx.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                    //display first 8 profiles
+                    $("#teamContent").html(content);
+
+                    //hide button
+                    $("#btnShowMore").hide();
+                }
+            }
+        });
+    }
+</script>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/past-scholarx.html]: This script seems unused. Remove it.

File: src/main/webapp/scholarx/past-scholarx.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                    <h5 class="title">
+                        <span class="d-block mb-1">{{name}}</span>
+                        <!--<small class="h6 text-muted">{{intro}}</small>-->
+                    </h5>
+                </div>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/past-scholarx.html]: This template seems unused. Remove it.

File: src/main/webapp/scholarx/past-scholarx.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/esm/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
+<!-- Optional JS -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
+<!-- Argon JS -->
+<script src="/assets/js/argon.js?v=1.0.1"></script>
+<script src="/assets/js/global-custom-js.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/past-scholarx.html]: Unused dependency

File: src/main/webapp/scholarx/past-scholarx.html [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+          content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description"
+          content="ScholarX is an exclusive 3-month program aimed at providing mentoring support to
+                            a selected pool of High Potential undergraduate students based in Sri Lanka ideally by a Sri Lankan expat currently engaged with one of the world’s top universities or Fortune 500 companies. It’s our free premium mentoring platform by Sri Lankans for Sri Lankans working
+                            towards creating a culture of knowledge and expertise sharing without the limitation of geographical borders.">
+    <meta name="author"
+          content="Sustainable Education Foundation">
+    <meta property="og:image"
+          content="assets/img/brand/scholarx.png">
+    <title>ScholarX | Sustainable Education Foundation</title>
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/past-scholarx.html]: I guess the title should be changed

File: src/main/webapp/scholarx/past-scholarx.html [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+        <div class="row">
+            <div class="col-md-6">
+                <img class="card-img"
+                     src="images/card-image.jpg">
+                <div class="card">
+                    <div class="card-body">
+                        <h5 class="card-title">ScholarX 2019</h5>
+                        <p class="card-text">For the 2019 cohort, we had 6 exceptional mentors based in the UK, US,
+                            Canada and Australia who generously volunteered their time.</p>
+                        <a href="../scholarx/archives/index.html"
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/past-scholarx.html]: `<a href="2019"`

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Good job @anjula-sack !

Check the following suggestions,
- [x] Do not touch `argon.css` file
- [x] There's an already created page for archives; `/scholarx/archives/index.html` use that one.
- [x] Link 2019 card with `/scholarx/archives/2019`
- [ ] Create a file names `styles.css` in the same directory if you need to write custom CSS .
- [x] Move images  to  `/scholarx/archives/images/`
- [x] Check other requested changes.
- [x] Fix the preview link. Link it to the archive pages

File: src/main/webapp/assets/css/argon.css (hunk 1 above)
Comment (jayasanka-sack) [reply src/main/webapp/assets/css/argon.css]: Now you have to edit `styles.css` file in the same directory. If that rule is used multiple files. you have to move them for the immediate parents style.css file.

File: src/main/webapp/scholarx/past-scholarx.html (hunk 2 above)
Comment (jayasanka-sack) [reply src/main/webapp/scholarx/past-scholarx.html]: @Gravewalker666  Now you have to edit styles.css file in the same directory. If that rule is used multiple files. you have to move them for the immediate parents style.css file.

File: src/main/webapp/scholarx/past-scholarx.html (hunk 3 above)
Comment (anjula-sack) [reply src/main/webapp/scholarx/past-scholarx.html]: Okaa

File: src/main/webapp/scholarx/archive/style.css [hunk 8]
```diff
@@ -0,0 +1,4 @@
+.card-img{
+    height : 140px;
+    object-fit : cover;
+}
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/archive/style.css]: EOF

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — APPROVED** [review]

**Review by jayasanka-sack — APPROVED** [review]
Thanks

**Pull request merged**